docker-compose.yml

```diff
@@ -42,3 +42,13 @@ services:
       - ./database/snapshots:/snapshots
     ports:
       - 5432:5432
+
+  cr3-user-api:
+    build: api
+    # image: atddocker/vz-user-cr3-api
```

Comment:
is this here as an option for us to uncomment when running locally? or will it be uncommented before merging